mesop/components/bokeh_plot/bokeh_plot.py

```diff
@@ -0,0 +1,29 @@
+from io import BytesIO
+from typing import Protocol
+from bokeh.plotting import figure
```

Comment:
can we remove this? it looks like it's not needed because figure is redefined below.